force-app/main/default/triggers/opportunityTrigger.trigger

```diff
@@ -0,0 +1,52 @@
+trigger opportunityTrigger on Opportunity (before update, before delete) {
+    // 1. Opportunity Update - Validate Amount > 5000
+    if (Trigger.isUpdate) {
+        for (Opportunity opp : Trigger.new) {
+            if (opp.Amount < 5000) {
```

Comment:
```suggestion
            if (opp.Amount <= 5000) {
```
The requirements say GREATER than 5000, so if amount is 5000, that should be an error as well.